### PR TITLE
Update to latest nightly .NET Core SDK

### PR DESCRIPTION
**Pull request by JamesNK**

### CODE REVIEW

**Comment by JunTaoLuo** [comment]
haha I'm making the same changes locally.

**Comment by JunTaoLuo** [comment]
How are you not running into the error ` MSB3822: Preserialized resources require the System.Resources.Extensions assembly at runtime, but it was not found in this project's references.`

**Comment by JunTaoLuo** [comment]
Looks like you are

**Comment by JamesNK** [comment]
> Looks like you are

Not locally 🤷‍♂

File: examples/Server/TicketRepository.cs [hunk 1]
```diff
@@ -35,7 +35,7 @@ public int GetAvailableTickets()
             return _availableTickets;
         }
 
-        public bool BuyTickets(string user, int count)
+        public bool BuyTickets(string? user, int count)
```
Comment (JunTaoLuo) [inline examples/Server/TicketRepository.cs]: Looks like we resolved the nullable issue differently

Comment (JamesNK) [inline examples/Server/TicketRepository.cs]: Undo this

Comment (JamesNK) [reply examples/Server/TicketRepository.cs]: There will always be a user with a name because the method forces authorization. My fix is better 😋

**Pull request merged**